substrate/frame/staking-async/src/pallet/impls.rs

```diff
@@ -1942,7 +1949,13 @@ impl<T: Config> Pallet<T> {
 		// Sanity check for the paged exposure of the active era.
 		let mut exposures: BTreeMap<T::AccountId, PagedExposureMetadata<BalanceOf<T>>> =
 			BTreeMap::new();
-		let era = ActiveEra::<T>::get().unwrap().index;
+		// If the pallet is not initialized, we return immediately from pallet's do_try_state() and
+		// we don't call this method. Otherwise, Eras::do_try_state enforces that both ActiveEra
+		// and CurrentEra are Some. Thus, we can safely unwrap here.
+		let era = ActiveEra::<T>::get()
+			.expect("ActiveEra must be set when checking paged exposures")
```

Comment:
maybe nit (I know it is just try-runtime, but :)):
```suggestion
			.ok_or(TryRuntimeError::Other("ActiveEra must be set when checking paged exposures"))?
```

Reply:
Not a hill I want to die on but: 

1. While I personally prefer `.unwrap()` / `expect()` in such code paths
2. We do have `-> Result<(), ()>` in the `try_state` code path, so I'd respect the overarching return value and do my best to return errors rather than panic.